src/day1.js

```diff
@@ -1,48 +1,49 @@
+import './App.css';
 export const Day1 = () => {
     return <div className="day1">
 
         <div>
             <p>I have installed NVM by running the following command on the terminal
         </p>
-            <p>wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+            <p className="code">wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
```

Comment:
instead of this use <Code>wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash</Code>